xivsupport/src/main/java/gg/xp/xivsupport/gui/map/MapPanel.java

```diff
@@ -711,6 +711,8 @@ else if (cbt.getType() == CombatantType.FAKE || hp.max() == 1 || hp.current() <
 			else {
 				hpBar.setVisible(false);
 			}
+
+			idLabel.setText((displayIds.get() ? String.format("0x%X", cbt.getId()) : ""));
```

Comment:
```suggestion
			idLabel.setVisible(displayIds.get());
```
setVisible should be faster since it performs less operations if there is no change in state.